src/app/zap-templates/zcl/data-model/chip/push-av-stream-transport-cluster.xml

```diff
@@ -0,0 +1,216 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/PushAVStreamTransport.adoc
+Parameters: in-progress 
+Git: 0.7-fall2024-ncr-1158-g413e195c2
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="General"/>
+  <bitmap name="PushAVStreamTransportSupportedContainerFormatsBitmap" type="bitmap8">
```

Comment:
This does not match the name in the spec (which is just SupportedContainerFormatsBitmap).

Stopping here until the difference wrt spec is clarified.

Reply:
Doh, sincere apologies, had started this back in October & forgot to regenerate with latest spec.

Updated